webapp/app/templates/components.html

```diff
@@ -279,3 +279,11 @@ <h5 class="modal-title mb-n2">{{title}}</h5>
         </a>
     </div>
 {% endmacro %}
+
+{% macro eligibilitySuccessBox(heading, textButtonOne, textButtonTwo, urlButtonOne, urlButtonTwo, targetButtonOne, targetButtonTwo , sourceButtonOne, sourceButtonTwo, plausible_domain_set=False, float_right=False) -%}
+    <h2>{{heading}}</h2>
+    <div class="eligibilitySuccessButtons">
+        <a class="{% if  float_right %} float-right {% endif %} btn btn-primary" href="{{ urlButtonOne }}" onclick="plausible('{{ targetButtonOne }}', {props: {method: '{{ sourceButtonOne }}'}})">{{ textButtonOne }}</a>
```

Comment:
do you need float_right? can't find a reference

Reply:
good point.